docs/source/workflows/predictors.rst

```diff
@@ -125,4 +125,9 @@ The following example demonstrates how to use the python SDK to create a :class:
    )
 
    # register predictor
-   predictor = project.predictors.register(shear_modulus_predictor)
\ No newline at end of file
+   predictor = project.predictors.register(shear_modulus_predictor)
+
+Predictor Reports
+--------------------
+
+A :doc:`predictor report <predictor_reports>` contains information about a machine-learned model and various performance metrics. 
```

Comment:
A predictor report _describes_ the model. What were its settings, what features are important to the model, etc. The report specifically does _not_ provide performance metrics. For that, you want to point to PerformanceWorkflows.

Reply:
Good to know - I actually just copied that from the first line of the existing predictor report doc. I will make an update in both locations.

Reply:
@sparadiso tried to clarify this, please review